src/shared/components/app/navbar.tsx

```diff
@@ -240,7 +244,7 @@ export class Navbar extends Component<NavbarProps, NavbarState> {
               </NavLink>
             </li>
             {amAdmin() && (
-              <li className="nav-item">
+              <li className="nav-item nav-item-admin">
```

Comment:
These admin-item-x classes seem unique. Maybe they're better off as IDs?

Reply:
Fair point, I'll switch up the nav items now